warehouse/utils/utils_test.go

```diff
@@ -18,6 +19,20 @@ import (
 	. "github.com/rudderlabs/rudder-server/warehouse/utils"
 )
 
+func TestDestinationConfigKeys(t *testing.T) {
+	for _, whType := range WarehouseDestinations {
+		t.Run(whType, func(t *testing.T) {
+			require.Contains(t, WHDestNameMap, whType)
+
+			whName := WHDestNameMap[whType]
+			configKey := fmt.Sprintf("Warehouse.%s.feature", whName)
+			got := config.TransformKey(configKey)
```

Comment:
This unit test is testing only func transformKey. Looks redundant to apply to WHDestNameMap.